Source/WebInspectorUI/UserInterface/Views/ConsoleGroup.js

```diff
@@ -79,6 +81,36 @@ WI.ConsoleGroup = class ConsoleGroup extends WI.Object
     {
         this._messagesElement.appendChild(messageOrGroupElement);
     }
+    
+    // FIXME: <https://webkit.org/b/264489> We should merge this logic with `WI.ConsoleMessageView.prototype.render`
+    setMessageLevel(messageLevel)
+    {
+        if (!this._element)
+            console.assert(this._element);
```

Comment:
sorry i wasnt entirely clear.  i was suggesting we get rid of the `if` as well

in this case we actually _have_ to get rid of the `if` as `console.*` are stripped from production builds, meaning we'd have a dangling `if`

```
setMessageLevel(messageLevel)
{
    console.assert(this._element);

    switch (messageLevel) {
    ...
    }
}
```

Reply:
TIL! Fixed in b59a15fd7276bb164a3b618611ab4ce7b6009788.

Thanks for all of the feedback. I learned a lot in the process!